src/index.js

```diff
@@ -331,7 +331,7 @@ module.exports = (
     }
 
     for (const chunk of stats.compilation.chunks) {
-      finalizeOutput(chunk.files[0]);
+      finalizeOutput(chunk.files.values().next().value);
```

Comment:
There's a merge conflict here after reverting the API in the other PR.

In fact, there is no `finalizeOutput()` function anymore 🤷‍♂

Reply:
Just rebased.